composer.json

```diff
@@ -59,6 +59,7 @@
         "phpstan/phpdoc-parser": "^2.0.0",
         "predis/predis": "^2.2.2",
         "privacyportal/oauth2-privacyportal": "^0.1.1",
+        "runtime/frankenphp-symfony": "*",
```

Comment:
Normally, for reproducible its a good practice to pin the package to at least a major version like `^0.2.0`